opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -100,7 +100,7 @@ protected void onSpanFinished(MockSpan mockSpan) {
     }
 
     @Override
-    public SpanBuilder buildSpan(String operationName) {
+    public Tracer.SpanBuilder buildSpan(String operationName) {
```

Comment:
`mockTracer.buildSpan("parent").start()` fails in packages different what `io.opentracing.mock` because mock `SpanBuilder` has package visibility.

There are two options: 
* change visibility of mock `SpanBuilder` to public
* `MockTracer#buildSpan` return `Tracer.SpanBuilder`